script.js

```diff
@@ -1,4 +1,307 @@
+// The game starts with Player 1.
+var currPlayer = 1;
+
+// Keep track of each player's dice rolls
+var player1Dice = [];
+var player2Dice = [];
+
+// Keep track of each player's details
+var playerProfiles = [
+  {id: 1, diceNum: 0, score: 0},
+  {id: 2, diceNum: 0, score: 0},
+];
+
+/*====================================
+==============HELPER FUNCTIONS========
+======================================*/
+/**
+ * Return a random number from 1 to 6
+ */
+var getDiceRoll = function () {
+  return Math.ceil(Math.random() * 6);
+};
+
+/**
+ * Get dice rolls for curr player and populate curr player's dice array
+ * Return the new dice rolls
+ */
+var getDiceRolls = function () {
+  // Create an array newDiceRolls with 2 independent dice roll values
+  var newDiceRolls = [];
+  for (var i = 0; i < numDiceChosen; i += 1) {
+    newDiceRolls.push(getDiceRoll());
+  }
+
+  // Assign newDiceRolls to the current player's dice array
+  if (currPlayer === 1) {
+    player1Dice = newDiceRolls;
+  }
+  // If currPlayer is not 1, assume currPlayer is 2
+  else {
+    player2Dice = newDiceRolls;
+  }
+
+  // Return new dice rolls to parent function
+  return newDiceRolls;
+};
+
+/**
+ * Return a number that is the concatenation of num1 and num2
+ * @param {number} num1
+ * @param {number} num2
+ */
+var concatenate2Numbers = function (num1, num2) {
+  return Number(String(num1) + String(num2));
+};
+
+/**
+ * Sort an array such that it can order any number of elements
+ * @param {Array} anArray
+ * @returns {Array} anArray sorted in ascending order
+ */
+var sortAnArray = function (anArray) {
+  // get the length of the given array
+  var length = anArray.length;
+
+  //Use bubble sort to sort a given array
+  // More info on bubble sort: https://www.geeksforgeeks.org/bubble-sort/
+  for (var i = 0; i < length - 1; i += 1) {
+    for (var j = 0; j < length - 1; j += 1) {
+      if (anArray[j] > anArray[j + 1]) {
+        var temp = anArray[j];
+        anArray[j] = anArray[j + 1];
+        anArray[j + 1] = temp;
+      }
+    }
+  }
+  return anArray;
+};
+
+/**
+ * Automate choosing the player's number from the dice rolls
+ * Return the player number
+ * @param {number} firstNumeralIndex
+ */
+var getPlayerNumberAutomatically = function () {
+  // Get the current player's dice array.
+  // We use the ternary operator on the following line for more concise syntax.
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator
+  // The example below is equivalent to:
+  // var diceArray;
+  // if (currPlayer === 1) { diceArray = player1Dice; } else { diceArray = player2Dice; }
+  var diceArray = currPlayer === 1 ? player1Dice : player2Dice;
+  var playerNum;
+  // Sort the array such that it can order any number of elements in ascending order
+  sortAnArray(diceArray);
+  //Alternatively, instead of writing our own sorting algorithm to sort arrays, we can use a native native JS method: diceArray.sort()
```

Comment:
Spacing after `//`. Consistency is key. Prettier should take care of this, please use prettier formatting for Basics.